OVERVIEW.md

```diff
@@ -0,0 +1,265 @@
+# Package Overview
+
+The geomstats package essentially consists of three layers: 
++ the [backend](geomstats/backend) module 
+providing an agnostic interface to numpy, pytorch and tensorflow,
++ the [geometry](geomstats/geometry) module 
+implementing a variety of geometric structures, 
++ the [learning](geomstats/learning) module
+adapting usual optimisation algorithms to manifolds.
```

Comment:
"usual learning algorithms to manifolds": pymanopt/geoopt do the optimization, we do the learning wrapper that mimics scikit-learn.

Reply:
Changed that. An overview of this package would be welcome as well, is K-means and PCA indeed learning? should we refer to pymanopt for their optimisation layer?

Reply:
K-means and PCA fall under the 'learning' umbrella indeed